docs/content/docker-images/client.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Porter Client Docker Image
+description: How to use the getporter/porter docker image
```

Comment:
nit: here and in `workshop.md`: should we capitalize `Docker` when not specifically referencing the `docker` cli name?  Perhaps also Porter when not the `porter` cli by name?

Reply:
haha I never know what to do about that! I can do a sweep on these two files but I'm sure we are inconsistent everywhere.